repos/system_upgrade/el7toel8/actors/checksendmail/libraries/library.py

```diff
@@ -0,0 +1,25 @@
+import os
+import re
+import glob
+
+SendmailConfDir = '/etc/mail'
+SendmailConfFiles = ['/etc/mail/sendmail.cf', '/etc/mail/sendmail.mc', '/etc/mail/submit.cf', '/etc/mail/submit.mc']
+# false positives blacklist
+rfp = re.compile(r'(^\s*RIPv6:::1\b)|(@\s+\[IPv6:::1\]\s+>)')
+
+def check_false_positives(f, l):
+    return f in ["/etc/mail/sendmail.cf", "/etc/mail/submit.cf"] and rfp.search(l) is not None
+
+def check_files_for_compressed_ipv6():
+    migrate_files = []
+    g = glob.glob(SendmailConfDir + '/*.db')
+    files = list(map(lambda x: re.sub(r'\.db$', '', x), g)) + SendmailConfFiles
+    regex = re.compile(r'IPv6:[0-9a-fA-F:]*::')
+    for filename in files:
+        if os.path.exists(filename):
```

Comment:
I'd use 
```
if not os.path.exists(filename):
    continue
```
This will decrease the nesting by one and make it easier to follow
Or add filtering as a list comprehension at L18
`for filename in [f for f in files if os.path.exists(f)]`

Reply:
Fixed